k8s/apply_infra.sh

```diff
@@ -85,10 +103,17 @@ echo "-----------------------------------"
 
 kubectl apply -f "$(dirname "$0")/frontend-configs/dust-frontend-config.yaml"
 
+echo "-----------------------------------"
+echo "Applying HPAs"
+echo "-----------------------------------"
+
+kubectl apply -f "$(dirname "$0")/hpas/apache-tika-hpa.yaml"
```

Comment:
Actually this needs to be applied after deployments because it references the deployment (so it's going to fail)

Reply:
nvm me already fixed 👍